docs-new/home/roadmap.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Roadmap"
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Roadmap Overview
+
+This roadmap outlines projects that the Iceberg community is working on, their priority, and a rough size estimate.
+This is based on the latest [community priority discussion](https://lists.apache.org/thread.html/r84e80216c259c81f824c6971504c321cd8c785774c489d52d4fc123f%40%3Cdev.iceberg.apache.org%3E).
+Each high-level item links to a Github project board that tracks the current status.
+Related design docs will be linked on the planning boards.
+
+# Priority 1
+
+* API: [Iceberg 1.0.0](https://github.com/apache/iceberg/projects/3) [medium]
```

Comment:
Looks like this is out of date. Iceberg 1.0 was released a long time ago. Do we have a more recent copy?

Reply:
We haven't merged it yet. I need to get back to that one. Perhaps we can get it added to 1.4.0? Otherwise I'd like to avoid changing content yet.